Recast/Cells/ContinueCollectionViewCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ContinueListeningCollectionViewCell.swift
+//  Recast
+//
+//  Created by Jaewon Sim on 9/19/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ContinueCollectionViewCell: UICollectionViewCell {
+    private var seriesImageView: UIImageView!
+    private var titleLabel: UILabel!
+    private var detailLabel: UILabel!
+    private var timeLeftLabel: UILabel!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .black
+
+        seriesImageView = UIImageView()
+        seriesImageView.layer.cornerRadius = 8
```

Comment:
can you make this corner radius a constant? we're probably going to use this in a lot of image view locations